### PR TITLE
Update change logs

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
Ready

## Description
updated change logs after release 19.8.0.
after running the `release_notes_clear.py` script, i went over the changes to make sure what was out in the release (so i updated wording and style according to final release notes)

## Required version of Demisto
19.8.0

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

File: Integrations/PaloAltoNetworks_XDR/CHANGELOG.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 ## [Unreleased]
+
+
+## [19.8.0] - 2019-08-06
 Added instructions in the integration instance Detailed Description section how to generate an API Key, API Key ID, and how to copy the integration URL.
```
Comment (yardensade) [inline Integrations/PaloAltoNetworks_XDR/CHANGELOG.md]: This does not seem to be part of the actual RN, will take a look in a few.

File: Integrations/integration-Cofense-Intelligence_CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
 ## [Unreleased]
+
+
+## [19.8.0] - 2019-08-06
 - Updated the integration name to Cofense Intelligence.
```
Comment (yardensade) [inline Integrations/integration-Cofense-Intelligence_CHANGELOG.md]: cofense was new integration in 19.8.0 release

**Comment by yardensade** [comment]
AWS - Lambda was new integration in 19.8.0, does not appear here.
```
AWS - Lambda
Amazon Web Services Serverless Compute service (lambda).
```

File: Integrations/Uptycs/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
 ## [Unreleased]
+
+
+## [19.8.0] - 2019-08-06
 #### New integration Uptycs
```
Comment (yardensade) [inline Integrations/Uptycs/CHANGELOG.md]: Seems to only appear in uptycs, not sure if should be in other 2 new integrations as well or not here.

File: Integrations/integration-PhishMe_CHANGELOG.md [hunk 4]
```diff
@@ -1,2 +1,5 @@
 ## [Unreleased]
+
+
+## [19.8.0] - 2019-08-06
 Deprecated. Use the Cofense Intelligence integration instead.
```
Comment (yardensade) [inline Integrations/integration-PhishMe_CHANGELOG.md]: Do we curently know how to treat deprecated integrations? in last RN had to due it manually

File: Scripts/script-CBSearch_CHANGELOG.md [hunk 5]
```diff
@@ -1,2 +1,5 @@
 ## [Unreleased]
-Deprecated. Use the ***cb-binary*** command and the ***cb-get-processes*** command, instead.
\ No newline at end of file
+
+
+## [19.8.0] - 2019-08-06
+Deprecated. Use the ***cb-binary*** command and the ***cb-get-processes*** command instead.
```
Comment (yardensade) [inline Scripts/script-CBSearch_CHANGELOG.md]: didnt put it as deprecated either, had to do it manually.

**Comment by yardensade** [comment]
Also `Uptycs - Bad IP Incident` new playbook is missing.

File: Integrations/Uptycs/CHANGELOG.md (hunk 3 above)
Comment (yaakovi) [reply Integrations/Uptycs/CHANGELOG.md]: this was the only one with release notes so i left it...
we never forced release notes for new content entities.

**Pull request merged**